api/api-doc/raft.json

```diff
@@ -62,6 +62,38 @@
                ]
             }
          ]
```

Comment:
One potential problem with clean-up is that it might cause CI to run 100 repetitions of multiple tests, killing the run (or timing it out).

So we might need to do the follow-up in batches -- and multiple PRs will be required in that case, or perhaps somehow bypassing CI / making it run fewer repetitions.

To make the work more incremental, I'd actually prefer separate PR(s) for this follow-up.

Reply:
Created the #19662 for the follow-up cleanup.